src/deluge/processing/engines/audio_engine.cpp

```diff
@@ -438,6 +436,17 @@ inline void cullVoices(size_t numSamples, int32_t numAudio, int32_t numVoice) {
 		if (numSamplesOverLimit >= 10) {
 			D_PRINTLN("under min voices but culling anyway");
 			cullVoice(false, FORCE, numSamples, nullptr);
+			cullType = CPUCullingIndicator::FORCE;
+		}
+	}
+	// blink LED to alert the user
+	if ((FlashStorage::defaultCPUCullingIndicatorMode != CPUCullingIndicator::OFF)
+	    && (cullType != CPUCullingIndicator::OFF)) {
+		if ((FlashStorage::defaultCPUCullingIndicatorMode == CPUCullingIndicator::ALL)
+		    || (FlashStorage::defaultCPUCullingIndicatorMode == cullType)) {
```

Comment:
this means that hard culls won't blink if you have it set to soft, which is probably not desirable